fish_completion/kerl.fish

```diff
@@ -0,0 +1,143 @@
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a install -d "Install the specified release at the given location"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a build-install -d "Builds and installs the specified release or git repository at the given location"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a deploy -d "Deploy the specified installation to the given host and location"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a update -d "Update the list of available releases from your source provider"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a list -d "List releases, builds and installations"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a delete -d "Delete builds and installations"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a path -d "Print the path of a given installation"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a active -d "Print the path of the active installation"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a plt -d "Print Dialyzer PLT path for the active installation"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a status -d "Print available builds and installations"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a prompt -d "Print a string suitable for insertion in prompt"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a cleanup -d "Remove compilation artifacts (use after installation)"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a emit-activate -d "Print the activate script"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a upgrade -d "Fetch and install the most recent kerl release"
+complete -c kerl -n "not __fish_seen_subcommand_from $kerl_args" -a version -d "Print current version"
+
+
+## build subcommand
+complete -c kerl -n "__fish_seen_subcommand_from build" -a "(kerl_build_complete)"
+
+## install command
+complete -c kerl -n "__fish_seen_subcommand_from install && not __fish_seen_argument (kerl_otp_builds)" -a "(kerl_otp_builds)"
+complete -c kerl -n "__fish_seen_subcommand_from install && __fish_seen_argument (kerl_otp_builds)" -a "(__fish_complete_directories)"
+
+## build-install subcommand
+complete -c kerl -n "__fish_seen_subcommand_from build-install && not __fish_seen_argument git (kerl_otp_releases)" -a "git (kerl_otp_releases)"
+complete -c kerl -n "__fish_seen_subcommand_from build-install && __fish_seen_argument git (kerl_otp_releases)" -a "(__fish_complete_directories)"
+
+## delopy subcommand
+complete -c kerl -n "__fish_seen_subcommand_from deploy" -a "(kerl_deploy_complete)"
+
+## update subcommand
+complete -c kerl -n "__fish_seen_subcommand_from update && not __fish_seen_argument releases" -a "releases" -d "All available OTP releases"
```

Comment:
A similar `All available OTP releases` exists for `list releases` but I don't see it here.

Reply:
Do you mean that the completion works, but its description does not show up?

Reply:
I didn't test the completion, but the description doesn't seem to show up in the code, directly. i.e., `All available OTP releases` exists in `bash_completion`, but not here, so I'm maybe missing some detail.

Reply:
I'll merge as-is, since the completion scripts (e.g. like the probes for installed packages) are mostly a user community-driven effort. If you feel something is missing, feel free to push another pull request; otherwise this is already a very good update.